sample-apps/django_sample/Procfile

```diff
@@ -2,4 +2,5 @@
 proxy: npx judoscale-adapter-proxy-server
 
 # Run with --noreload to avoid multiple server processes (and multiple Judoscale reporters)
-web: python manage.py runserver $PORT --noreload
+web: gunicorn django_sample.wsgi:application --preload
+#web: python manage.py runserver $PORT --noreload
```

Comment:
@adamlogic shall I guess here should default to django's runserver than the gunicorn?

Reply:
I was thinking we would use gunicorn instead of runserver. Do you think there's benefit to testing with runserver?

Reply:
runserver is the native django's command to serve the application locally. Gunicorn is one option to serve django applications, [uwsgi](https://uwsgi-docs.readthedocs.io/en/latest/Configuration.html) is another popular option. Both are implemented usually for production. If the sample app is meant to be deployed in heroku by your clients to try out, then I'd leave the gunicorn.

Reply:
> If the sample app is meant to be deployed in heroku by your clients to try out

Not really. It's more for us (developers working on the adapter) to verify that the adapter is working as expected. So it's not really designed to be deployed to Heroku, but I do want it to _simulate_ how Django apps are commonly run on Heroku. So we'll leave the unicorn. :-)